### PR TITLE
Include isCrossDevice in capture payload sdk_metadata

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
The `sdk_metadata` payload sent to API endpoints for Document, Face captures is missing the `isCrossDevice` property. This stops us from being able to gather data for questions like performance of cross device uploads versus non cross device.

# Solution
Include `mobileFlow` property to be passed down to current component as existing `isCrossDevice` property in Document, Face capture payload of `sdk_metadata` actually relies on this property's value.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1235-pr-onfido-sdk-ui-onfido.surge.sh

**Review by eraxyz — APPROVED** [review]

File: src/components/Router/StepsRouter.js [hunk 1]
```diff
@@ -65,7 +65,13 @@ class StepsRouter extends Component {
...
         >
           <CurrentComponent
-            {...{ ...options, ...globalUserOptions, ...otherProps, back }}
+            {...{
+              ...options,
+              ...globalUserOptions,
+              ...otherProps,
+              mobileFlow,
+              back,
+            }}
```
Comment (ethan605) [inline src/components/Router/StepsRouter.js:74]: I'm against this usage, nested rest operators seem confusing especially in several lines.
```suggestion
          const passedProps = {
              ...options,
              ...globalUserOptions,
              ...otherProps,
              mobileFlow,
              back,
            }
          {...passedProps}
```

**Review by ethan605 — APPROVED** [review]
LGTM, just a minor comment on passing props.

File: src/components/Router/StepsRouter.js (hunk 1 above)
Comment (kopijunkie) [reply src/components/Router/StepsRouter.js:74]: Great suggestion. Think part of why this property got missed out was from the use of nested rest operators around the code here in general.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1235-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1235-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**